components/pepper_flash/BUILD.gn

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2018 GitHub, Inc.
+# Use of this source code is governed by the MIT license that can be
+# found in the LICENSE file.
+
+component("pepper_flash") {
+  visibility = [ "//electron:electron_lib" ]
+  defines = [ "IS_PEPPER_FLASH_IMPL" ]
+  sources = [
+    "//chrome/browser/renderer_host/pepper/chrome_browser_pepper_host_factory.cc",
+    "//chrome/browser/renderer_host/pepper/chrome_browser_pepper_host_factory.h",
+    "//chrome/browser/renderer_host/pepper/pepper_broker_message_filter.cc",
+    "//chrome/browser/renderer_host/pepper/pepper_broker_message_filter.h",
+    "//chrome/browser/renderer_host/pepper/pepper_flash_browser_host.cc",
+    "//chrome/browser/renderer_host/pepper/pepper_flash_browser_host.h",
+    "//chrome/browser/renderer_host/pepper/pepper_flash_clipboard_message_filter.cc",
+    "//chrome/browser/renderer_host/pepper/pepper_flash_clipboard_message_filter.h",
+    "//chrome/browser/renderer_host/pepper/pepper_flash_drm_host.cc",
+    "//chrome/browser/renderer_host/pepper/pepper_flash_drm_host.h",
+    "//chrome/browser/renderer_host/pepper/pepper_isolated_file_system_message_filter.cc",
+    "//chrome/browser/renderer_host/pepper/pepper_isolated_file_system_message_filter.h",
+    "//chrome/renderer/pepper/chrome_renderer_pepper_host_factory.cc",
+    "//chrome/renderer/pepper/chrome_renderer_pepper_host_factory.h",
+    "//chrome/renderer/pepper/pepper_flash_drm_renderer_host.cc",
+    "//chrome/renderer/pepper/pepper_flash_drm_renderer_host.h",
+    "//chrome/renderer/pepper/pepper_flash_font_file_host.cc",
+    "//chrome/renderer/pepper/pepper_flash_font_file_host.h",
+    "//chrome/renderer/pepper/pepper_flash_fullscreen_host.cc",
+    "//chrome/renderer/pepper/pepper_flash_fullscreen_host.h",
+    "//chrome/renderer/pepper/pepper_flash_menu_host.cc",
+    "//chrome/renderer/pepper/pepper_flash_menu_host.h",
+    "//chrome/renderer/pepper/pepper_flash_renderer_host.cc",
+    "//chrome/renderer/pepper/pepper_flash_renderer_host.h",
+    "//chrome/renderer/pepper/pepper_helper.cc",
+    "//chrome/renderer/pepper/pepper_helper.h",
+    "//chrome/renderer/pepper/pepper_shared_memory_message_filter.cc",
+    "//chrome/renderer/pepper/pepper_shared_memory_message_filter.h",
```

Comment:
@deepak1556 Can some of these sources be replaces by dependencies?

Reply:
Unfortunately they aren't available as separate targets in `//chrome`, they are just build as a part of chrome's renderer and browser targets.